draft-ietf-quic-manageability.md

```diff
@@ -809,20 +816,83 @@ off the connection ID may therefore cause undetectable and unrecoverable loss
 of state in the middle of a connection. Use of connection ID specifically
 discouraged for NAT applications.
 
-## Passive Network Performance Measurement and Troubleshooting
+If a NAT hits an operational limit, it is recommended to rather drop the
```

Comment:
I think the caution against using Connection ID needs to be a lot stronger.  Things will break, users will notice.  I realize that implementations have to use this to enforce that, but I believe some already are.  See a comment below as well.

Reply:
Can you open a separate issue for this and propose some text (actually in the issue for now maybe, until this PR is merged) as this PR is only moving but not changing this part of the text.